### ISSUE TITLE
Makes the seekbar red again

### ISSUE THREAD

**Issue by SodaWithoutSparkles** [revanced/revanced-patches#2248]
### Application

Youtube

### Issue

When the player was not touched for a while, the seekbar becomes white instead of the usual red

### Patch

Make the seekbar red again

### Motivation

This returns the old look of the player. 

Besides, this should be controlled by the "change seekbar color" settings too.

![screenshot](https://cdn.discordapp.com/attachments/952946952348270626/1111177844027039744/Screenshot_2023-05-25-14-23-34-096_app.revanced.android.youtube-edit.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by aliernfrog** [comment]
Yeah, this is a stock YT feature. Seekbar is white when unfocused on dark theme.

The issue is seekbar color patch doesn't modify the color when "Seekbar color" option is set to red ("#FF0000").

To let users use stock YT seekbar colors, a toggle can be added, or the check for red color can be updated to check for empty string ("").

**Comment by getused2me** [comment]
taking FF0001 instead of FF0000 solved the issue for me. thanks.

**Comment by SodaWithoutSparkles** [comment]
> then choose any custom red color that is not 0xFF0000,
> such as 0xFF0001, or use any alpha channel with some amount of transparency (such as 0xEEFF0000)

Actually `#feff0000` would be fine, just decrease it by 1
